jekyll/_cci2/authentication.adoc

```diff
@@ -62,6 +62,43 @@ A user who logs in will be redirected to the Accounts page of the CircleCI appli
 
 NOTE: A user who has authenticated with LDAP and is then removed from LDAP/AD will be able to access CircleCI as long as they stay logged in (because of cookies). As soon as the user logs out or the cookie expires, they will not be able to log back in. A users' ability to see projects or to run builds is defined by their GitHub permissions. Therefore, if GitHub permissions are synced with LDAP/AD permissions, a removed LDAP/AD user will automatically lose authorization to view or access CircleCI as well.
 
+=== Known Issue
```

Comment:
LDAP workaround

Reply:
Thanks for sharing @rosieyohannan! When App Core wrote up this workaround in the Jira ticket, our thinking was it would probably make sense to direct users to customer support for this rather than including the detailed instructions, since they aren't very user friendly and muck around w/ site admin privileges in the database. It looks like this functionality has been broken for LDAP for a very long time (possibly years), and my understanding is that we have a relatively low number of Server customers spinning up LDAP installations, so this seems to be a pretty unlikely situation to run into. What do you think?

Reply:
Thanks @mzalimeni makes sense to me to just change this to a note then, telling people to contact support if they experience the issue. I'll check with the release team :-)

Reply:
We decided to keep the workaround in docs with a link from the changelog. I spoke to Michelle and Ryan A, who thought it would cause more sadness if people experience the issue and have to go to support if there are known steps already. Hope that's ok!

Reply:
@rosieyohannan I think that'd be fine. Since these are going into user-facing docs (and I'm now feeling hyper-vigilent about anything 2.19 related), I asked for a quick look from my team, and got a suggestion to improve clarity on a few things below. I'm going to drop a few more light edit suggestions if that's ok w/ you.

Reply:
Yes brilliant, thanks!